domapping/mapping.py

```diff
@@ -312,6 +316,24 @@ def __gen_type_properties(json_schema, path, resolver, config, es_mapping):
     return es_mapping
 
 
+def __guess_enum_type(enum_array, path):
```

Comment:
@nharraud do not use double underscore for making function "private".

--
https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles